Support/Multitenancy/Multiple-Ironic-conductors/ironic/dependencies.yaml

```
@@ -0,0 +1,17 @@
+dependencies:
+  - name: namespace
```

Use k8s indentation.
```suggestion
- name: namespace
```